### ISSUE TITLE
[Progression] Druid Swift Flying Form

### ISSUE THREAD

**Issue by balleny** [chromiecraft/chromiecraft#5104]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

70

### Current Behaviour

In wotlk druids can learn the swift flight form from their class trainers given the required riding skill.

### Expected Blizzlike Behaviour

during tbc progression, druids had to complete the quest chain starting with https://wowgaming.altervista.org/aowow/?quest=10955 to learn swift flight form

quest was released with phase 2 (t5 content) in tbc classic https://www.wowhead.com/tbc/quest=10955/morthis-whisperwing#comments:id=5244734

### Source

.

### Steps to reproduce the problem

.

### Extra Notes

https://wowgaming.altervista.org/aowow/?spell=40120#taught-by-npc

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/5b602508808a134e5df6051af7774952df00a7d0

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Gultask** [comment]
It's already in 😛
I will keep it here as part of a discussion, though

https://github.com/azerothcore/mod-progression-system/blob/main/src/Bracket_0/sql/world/progression_0_npc_trainer_flying.sql

**Comment by balleny** [comment]
> It's already in stuck_out_tongue I will keep it here as part of a discussion, though
> 
> https://github.com/azerothcore/mod-progression-system/blob/main/src/Bracket_0/sql/world/progression_0_npc_trainer_flying.sql

but not delayed to t5 content?

**Comment by Gultask** [comment]
nope, only until Heroics at the moment (exclusively because of Anzu)

**Comment by foo-17** [comment]
Tested in AzerothCore rev. 23606bcba0d2 2023-11-24 (master branch). Can confirm, quest "Morthis Whisperwing" (id: 10955) is offered by "Loganaar" (id: 12042) once necessary riding skill was purchased.

This issue can be closed.
